movement/movement_custom_signal_tunes.h

```diff
@@ -84,4 +84,21 @@ int8_t signal_tune[] = {
 };
 #endif // SIGNAL_TUNE_KIM_POSSIBLE
 
+#ifdef SIGNAL_TUNE_LAYLA
+int8_t signal_tune[] = {
+    BUZZER_NOTE_A6, 4,
+    BUZZER_NOTE_REST, 1,
+    BUZZER_NOTE_C7, 5,  
+    BUZZER_NOTE_REST, 2,
+    BUZZER_NOTE_D7, 5,
+    BUZZER_NOTE_REST, 1,
+    BUZZER_NOTE_F7, 5,
+    BUZZER_NOTE_REST, 1,
+    BUZZER_NOTE_D7, 5,
+    BUZZER_NOTE_REST, 2,
+    BUZZER_NOTE_D7, 15,
```

Comment:
In the song, the duration in the first repeat is dotted quarter (i.e. 30), in the second it's plain quarter (i.e. 20), so you may want to make it at least 20.

Reply:
got that 👍